e2e/commons-test.mk

```diff
@@ -0,0 +1,74 @@
+FEATURES?=
+TAGS?=
+DEVELOPER_MODE?=false
+ELASTIC_APM_ACTIVE?=false
+FORMAT?=pretty
+LOG_INCLUDE_TIMESTAMP?=TRUE
+LOG_LEVEL?=INFO
+TIMEOUT_FACTOR?=3
+#true by default, allowing developers to set SKIP_SCENARIOS=false
+SKIP_SCENARIOS?=true
+STACK_VERSION?=
+
+ELASTIC_APM_ENVIRONMENT?=local
+
+ifeq ($(ELASTIC_APM_ACTIVE),true)
+ifeq ($(ELASTIC_APM_ENVIRONMENT),ci)
+JENKINS_STATS_SECRET?=secret/observability-team/ci/jenkins-stats
+export APM_SECRET_TOKEN?=$(shell vault read -field apmServerToken "$(JENKINS_STATS_SECRET)")
+export APM_SERVER_URL?=$(shell vault read -field apmServerUrl "$(JENKINS_STATS_SECRET)")
+endif
+
+ifeq ($(ELASTIC_APM_ENVIRONMENT),local)
+export APM_SECRET_TOKEN?=
+export APM_SERVER_URL?=http://localhost:8200
```

Comment:
There are two colons here. Should this value be quoted?

Reply:
I noticed that Makefiles behaves differently than shell scripts, and treats double quotes as part of the string 🤷‍♀️ 

>$ SUITE="fleet" TAGS="fleet_mode_agent && nightly" TIMEOUT_FACTOR=3 LOG_LEVEL=TRACE BEATS_USE_CI_SNAPSHOT=false DEVELOPER_MODE=true ELASTIC_APM_ENVIRONMENT=local ELASTIC_APM_ACTIVE=true make -C e2e functional-test 
GOARCH=amd64 go get -v github.com/cucumber/godog/cmd/godog@v0.11.0
cd _suites/fleet && \
	OP_LOG_LEVEL=TRACE \
	OP_LOG_INCLUDE_TIMESTAMP=TRUE \
	TIMEOUT_FACTOR=3 \
	STACK_VERSION= \
	DEVELOPER_MODE=true \
	ELASTIC_APM_SERVICE_NAME="E2E Tests" \
	ELASTIC_APM_CENTRAL_CONFIG="false" \
	ELASTIC_APM_GLOBAL_LABELS= \
	ELASTIC_APM_SERVICE_VERSION="" \
	ELASTIC_APM_ENVIRONMENT="local" \
	ELASTIC_APM_SECRET_TOKEN="" \
	ELASTIC_APM_SERVER_URL="http://localhost:8200" \
	godog --format=pretty --tags "fleet_mode_agent && nightly && ~skip"
^Cmake: *** [functional-test] Interrupt: 2

Reply:
I think Make is special in this case in that it will interpret both `'` and `"` as literal